pyproject.toml

```
@@ -30,7 +30,7 @@ types-retry = "^0.1.5"
 tabulate = "^0.8.9"
 types-tabulate = "^0.8.3"
 sqlparse = "^0.4.2"
-cognite-sdk = "6.1.7"
+cognite-sdk = "6.1.8"
```

```suggestion
cognite-sdk = "6.1.7"
```
it looks like `6.1.7` is the latest one

---

SDK Version 6.1.8 is waiting to merge https://github.com/cognitedata/cognite-sdk-python/pull/1221